composer/core/logging/base_backend.py

```diff
@@ -12,9 +14,11 @@
     from composer.core.logging.logger import LogLevel, TLogData
     from composer.core.state import State
 
+__all__ = ["LoggerCallback"]
+
 
 class LoggerCallback(Callback, ABC):
-    """Base class for logging backends."""
+    """Base class for logger callback."""
```

Comment:
Can add a little more here, e.g. Loggers should extend this class, etc. This is a Callback with an additional interface for logging metrics, `log_metric`

Reply:
can also add something here on what types of data can be logged (e.g. TLogData is a bit vague)

Reply:
I just say `Data to be logged should be of the type :attr:`~.logger.TLogData`, but link to `TLogData`, which provides a more complete explanation.